ci/libvcx.dockerfile

```diff
@@ -6,11 +6,11 @@ WORKDIR /home/indy
 COPY --chown=indy  ./ ./
 
 USER indy
-ENV X86_64_ALPINE_LINUX_MUSL_OPENSSL_NO_VENDOR "true"
-RUN cargo build --release --manifest-path=/home/indy/Cargo.toml
+ENV X86_64_UNKNOWN_LINUX_MUSL_OPENSSL_NO_VENDOR "true"
+RUN cargo build --manifest-path=/home/indy/Cargo.toml
```

Comment:
I assume this was removed by mistake (part of development?). Or is there reason to build in debug?

Reply:
Yes, I removed it to be able to iterate faster (faster build speed with debug), will put release back